labels.yaml

```diff
@@ -2,4 +2,473 @@ orgs:
 - meltano
 - MeltanoLabs
 
-labels: []
+labels:
+  ####################
+  #      Colors      #
+  ####################
+  # red:            "B60205"
+  # orange:         "D93F0B"
+  # yellow:         "FBCA04"
+  # green:          "0E8A16"
+  # teal:           "006B75"
+  # blue:           "1D76DB"
+  # navy:           "0052CC"
+  # purple:         "5319E7"
+  # light_red:      "E99695"
+  # light_orange:   "F9D0C4"
+  # light_yellow:   "FEF2C0"
+  # light_green:    "C2E0C6"
+  # light_teal:     "BFDADC"
+  # light_blue:     "C5DEF5"
+  # light_teal:     "BFD4F2"
+  # light_purple:   "D4C5F9"
+
+  ####################
+  #      Common      #
+  ####################
+  - name: "Documentation"
+    description: Improvements or additions to documentation
+    color: "0075CA"
+    aliases:
+     - "documentation"
+  - name: "duplicate"
+    description: This issue or pull request already exists
+    color: "CFD3D7"
+  - name: "good first issue"
+    description: Good for newcomers
+    color: "7057FF"
+  - name: "help wanted"
+    description: Extra attention is needed
+    color: "008672"
+  - name: "invalid"
+    description: This doesn't seem right
+    color: "E4E669"
+  - name: "question"
+    description: Further information is requested
+    color: "D876E3"
+  - name: "wontfix"
+    description: This will not be worked on
+    color: "FFFFFF"
+  - name: "api"
+    color: "CD5B45"
+  - name: "Accepting Pull Requests"
+    color: "0E8A16"
+    aliases:
+      - "accepting pull requests"
+      - "accepting merge requests"
+      - "Accepting Merge Requests"
+  - name: "analyze"
+    color: "0033CC"
+  - name: "architecture decision"
+    color: "A8D695"
+  - name: "Awaiting Action/Author"
+    color: "9400D3"
+    aliases:
+      - "awaiting action::author"
+  - name: "backend"
+    color: "C39953"
+  - name: "breaking change"
+    color: "9400D3"
+  - name: "candidate"
+    color: "C39953"
+  - name: "CI/CD"
+    color: "E6E6FA"
+    aliases:
```

Comment:
Would you like this tool to auto-generate title-case and uppercase aliases automatically for all tags? That would be easy, and save us from adding such aliases manually.

Reply:
maybe. I've added a bunch in my latest commit.

Reply:
Logged here:
- https://github.com/meltano/.github/issues/7